src/main/java/com/maxmind/minfraud/request/TransactionReport.java

```diff
@@ -49,9 +49,6 @@ public static final class Builder {
          *                  fraudulent.
          */
         public Builder(InetAddress ipAddress, Tag tag) {
-            if (ipAddress == null) {
-                throw new IllegalArgumentException("ipAddress must not be null");
-            }
```

Comment:
I think we should add a new constructor that doesn't include `ipAddress` and then a builder method that sets it. This could then call that and we could deprecate this one.